tests/Common/AbstractMigratorTest.php

```diff
@@ -81,4 +83,24 @@ public function testGetMigrationNameLimitFromSchema(): void
 
         $this->assertGreaterThan(180, $migrator->getMigrationNameLimit());
     }
+
+    public function testMaxSqlOutputLength(): void
+    {
+        $informer = new StubMigrationInformer();
+
+        $migrator = new Migrator(
+            $this->container->get(ConnectionInterface::class),
+            $informer,
+            maxSqlOutputLength: 20,
+        );
+
+        $this->assertSame(20, $migrator->getMaxSqlOutputLength());
+
+        $migrator->up(new M231015155500ExecuteSql());
+
+        $this->assertStringContainsString(
+            'Execute SQL: CREATE TABLE person  [... hidden] ... Done',
```

Comment:
Seems, we can trim spaces after prepare SQL string length. What do you think? (If yes, we can make it in separate PR)

Reply:
Yes, looks good.

Reply:
https://github.com/yiisoft/db-migration/issues/214